Homework_5/task_3.py

```diff
@@ -0,0 +1,11 @@
+# 3
+with open('text.txt', 'r') as file:
+    data = file.read()
+
+results = set()
+for el in data.lower():
+    if el.isalpha():
+        results.update({f'Count of "{el}" is --> {data.count(el)}'})
```

Comment:
ok let it be. the solution is not optimal because you iterate through all the text. 

another solution is to use dict.
or import alphabet from string